code/modules/clothing/head/beanie.dm

```diff
@@ -88,3 +88,19 @@
 	icon_state = "beanierasta"
 
 //No dog fashion sprites yet :(  poor Ian can't be dope like the rest of us yet
+
+/obj/item/clothing/head/beanie/black/dboy
+	name = "test subject beanie"
+	desc = "A dingy and torn black beanie. Is that slime or grease?"
+	/// Used for the extra flavor text the d-boy himself sees
+	var/mob/living/carbon/human/owner = null
```

Comment:
this is going to hard delete

Reply:
`null` this on `dropped`

Reply:
What is it gonna delete? I've tested it and nothing seemed to delete.

Reply:
specifically that the hat never nulls the reference, (thus either leaving a hanging reference or forcing a harddel) mothblocks wants `dropped()` to make it null

Reply:
the point of the variable is to make it so even if someone else wears the beanie only the person who got it in the first place will have the message. Having it null when it's dropped kinda defeats the point.

I do have _another_ option I just thought of though

Reply:
yea nevermind I realized it would make no sense.
Shit!
I gotta figure something out for this then I guess!

Reply:
Just make this a weakref then and do `IS_WEAKREF_OF(examiner, owner)`

Reply:
idk what any of that means but if it works then I'll gladly do it, 1 sec